R/use_extendr.R

```diff
@@ -210,6 +230,11 @@ use_extendr <- function(path = ".",
     file.path("src", "rust", "vendor")
   )
 
+  # ensure that the target directory is ignored
+  usethis::use_build_ignore(
+    file.path("src", "rust", "target")
+  )
```

Comment:
ensure no accidental incorporation of target files into a bundle. Has no effect other than preventing it from being included in the final tarball